packages/react-components/src/components/AppFrame/components/NavigationTopBar/types.ts

```diff
@@ -12,16 +12,25 @@ export interface INavigationTopBarProps extends ComponentCoreProps {
   children?: React.ReactNode;
 
   /**
-   * Ńodes placed under the children.
+   * Nodes placed under the children.
    */
   additionalNodes?: React.ReactNode;
+
+  /**
+   * The CSS class name
+   */
+  className?: string;
```

Comment:
ComponentCoreProps already have the className prop, so it is unnecessary to define it here.